roles/keystone/templates/etc/apache2/sites-available/keystone.conf

```diff
@@ -1,17 +1,29 @@
 
 Listen {{ endpoints.keystone_admin.port.backend_api }}
 Listen {{ endpoints.keystone.port.backend_api }}
-{% macro setup_k2k_sp() -%}
-<Location /Shibboleth.sso>
-    SetHandler shib
-</Location>
-
-<LocationMatch /v3/OS-FEDERATION/identity_providers/.*?/protocols/saml2/auth>
-    ShibRequestSetting requireSession 1
-    AuthType shibboleth
-    ShibExportAssertion Off
-    Require valid-user
+{% macro setup_shibboleth() %}
+    <Location /Shibboleth.sso>
+        SetHandler shib
+    </Location>
```

Comment:
Yes, I wanted to increase readability and have the indentation match the rest of the file so I added indentation for inside tags. Also it seems I missed the spacing on the last tag so I'll update it.